app/i18n/webapp-de.json

```diff
@@ -1,7 +1,7 @@
 {
   "index.claim": "Sicherer Messenger für alle.",
   "index.createAccount": "Konto erstellen",
-  "index.createAccountFor": "für Privat",
+  "index.createAccountFor": "für den privaten Gebrauch",
```

Comment:
I asked Roger to take a look at this yesterday. Did the change come from him?

Reply:
I can't tell. We will have to ask him, if he changed anything on crowdin